### PR TITLE
Corrected grammar

### PR DESCRIPTION
**Pull request by DznDani**
It seems that the translation in spanish was made with a translator bc it has some grammatical errors, but its all fixed now, hope it helps

### CODE REVIEW

**Comment by Kataiser** [comment]
Yo thanks!

File: TF2 Rich Presence/locales/Spanish.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
     "4021625669": "(Si tiene más de una versión desactualizada, es posible que haya habido más cambios y correcciones que ésta.)",
-    "3336272700": "1. Haz clic con el botón derecho del ratón en Team Fortress 2 en tu biblioteca de Steam",
-    "3034581868": "2. Propiedades abiertas (muy abajo)",
-    "2781874761": "3. Haga clic en \"Establecer opciones de lanzamiento...\"",
-    "2276072520": "4. Añadir {0}",
-    "5268450630": "5. Aceptar y Cerrar",
-    "5334641500": "6. Reiniciar TF2",
+    "3336272700": "1. Haz click derecho en Team Fortress 2 en tu biblioteca de Steam",
+    "3034581868": "2. Abre propiedades",
+    "2781874761": "3. Haz click en \"Parametros de Lanzamiento...\"",
```
Comment (jmunoz94) [inline TF2 Rich Presence/locales/Spanish.json:9]: "Clic" is actually the correct version. Here you have a [Tweet clarifying that from RAEConsultas](https://twitter.com/raeinforma/status/1221741614177177600) and this is the [respective entry in the RAE dictionary](https://www.rae.es/dpd/clic).

File: TF2 Rich Presence/locales/Spanish.json [hunk 2]
```diff
@@ -19,21 +19,21 @@
...
     "9227519100": "Clase",
     "3231589050": "Clase: {0}",
-    "1351878611": "Cerrar sin grabar",
+    "1351878611": "Cerrar sin guardar",
     "5860897400": "Punto de Control",
-    "1795231764": "No pude encontrar una instalación de Team Fortress 2.",
+    "1795231764": "No se pudo encontrar una instalación de Team Fortress 2.",
     "3315339845": "No se pudieron cargar las configuraciones, volviendo a los valores por defecto.{0}",
-    "1791761380": "Accidentes",
+    "1791761380": "Crasheos",
```
Comment (jmunoz94) [inline TF2 Rich Presence/locales/Spanish.json:34]: "Crasheo" is an anglicism. "Accidentes" does not sound great either, but maybe something like "Fallos" sounds better and is more standard?

File: TF2 Rich Presence/locales/Spanish.json [hunk 3]
```diff
@@ -124,13 +124,13 @@
...
     "1589315666": "Recortar/limpiar console.log",
     "8988278560": "Modo de juego desconocido",
-    "1257244521": "Неизвестное имя сервера",
-    "2651621140": "Utilizar las imágenes clásicas del modo de juego en la interfaz gráfica de usuario",
+    "1257244521": "Nombre de Servidor desconocido",
+    "2651621140": "Utilizar las imágenes clásicas del modo de juego en la interfaz",
     "1011879672": "¿Desea abrir la página de descarga?",
     "3846993800": "Sí",
-    "1638865032": "Su instalación de TF2 aún no parece estar configurada correctamente. Para arreglar:",
+    "1638865032": "Su instalación de TF2 aún no parece estar configurada correctamente. Para arreglarla:",
```
Comment (jmunoz94) [inline TF2 Rich Presence/locales/Spanish.json:131]: This is using "usted" ("su") while other sentences use "tú" (such as the sentences in lines 7 to 12 in `TF2 Rich Presence/locales/Spanish.json`). For consistency this should also use "tú":

```suggestion
    "1638865032": "Tu instalación de TF2 aún no parece estar configurada correctamente. Para arreglarla:",
```

**Review by jmunoz94 — COMMENTED** [review]
Already merged, but I have some suggestions.

**Comment by DznDani** [comment]
> Already merged, but I have some suggestions.

Sorry, i did it kinda in a rush, and i thought i could use those words or anglicisms, idk if the owner would be ok with adding these changes but thanks for the corrections tho

**Comment by Kataiser** [comment]
I'm good with that yeah

**Pull request merged**